gui/src/Collaborators/components/projectMembers.tsx

```diff
@@ -25,6 +25,7 @@ import { connect } from 'react-redux';
 import '../styles/collaborators.scss';
 import { fetchMembersForProject } from '../actions/collaborators.actions';
 import { setCurrentProject } from '../../Projects/actions/projects.actions';
+import MouseEvent = __React.MouseEvent;
```

Comment:
It can, but for whatever reason Webstorm really wants to go with __React.MouseEvent anytime you use autocomplete. I pushed a third alternative that keeps things idiomatic and can direct Webstorm better. lmk which route you want to go with